spec/models/category_spec.rb

```diff
@@ -14,6 +14,13 @@
     expect(category).to_not be_valid
   end
 
-  xit "can have many events" do
+  it "can have many events" do
+    event1 = create(:event, title: "The Hobbit", description: "Hobbit movie 3")
+    event2 = create(:event, title: "Frozen", description: "movie about two sisters")
+
+    Categorization.create(category_id: category.id, event_id: event1.id)
+    Categorization.create(category_id: category.id, event_id: event2.id)
+    expect(category.events.count).to eq 2
+    expect(category.events.map(&:title)).to eq(["The Hobbit", "Frozen"])
```

Comment:
I really like the concept of making two events and adding them to a category and testing that there are two. 

Is there a way to use the factory girl factories for this? 

  it "can have many events" do
    category = create(:category) #creates a standard category using the details from factories.rb
    event1 = create(:event) #creates a standard event from factories.rb
    event2 = create(:event, title: "Frozen", description: "movie about two sisters")
    #creates a second event with the only differance the title and description
    category.events << event1
    category.events << event2
    #adds the events to the cateogries
    expect(category.events.count).to eq 2
    expect(category.events.map(&:title)).to eq(["Blazers vs. Clippers", "Frozen"])
    #Blazers vs. Clippers is the default title setup in the factories file
  end